packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-input.component.tsx

```diff
@@ -33,6 +35,23 @@ const VitalsBiometricInput: React.FC<VitalsBiometricInputProps> = ({
   inputIsNormal,
   isTablet,
 }) => {
+  const { t } = useTranslation();
+  const [invalid, setInvalid] = useState<boolean>(false);
+
+  function check(value) {
+    setInvalid(!(Number(value) || value === ''));
+  }
+  const error = () => {
+    return <FormLabel className={styles.danger}>{t('numericInputError', 'Must be a number')}</FormLabel>;
+  };
```

Comment:
Just inline this JSX in the return block. Avoid having JSX distributed around your components, when possible.